server/viame_tasks/tasks.py

```diff
@@ -1,16 +1,21 @@
 import json
 import os
-import re
 import tempfile
-from subprocess import Popen, PIPE
+from subprocess import Popen, PIPE, DEVNULL
+from pathlib import Path
```

Comment:
does `pathlib.Path` have some feature that `os.path` doesn't?  I tend to prefer the former for simple operations.

Reply:
`pathlib` supports more operations and direct handling of paths than `os.path`. With certain operations, `os.path` will just return a string, where as `pathlib` will return a valid `pathlib.Path` object.